packages/core/src/model/TransactionStatus.ts

```diff
@@ -1,11 +1,16 @@
 import { ChainId } from '../constants'
 import { TransactionResponse, TransactionReceipt } from '@ethersproject/abstract-provider'
 
+export type TransactionState = 'None' | 'Mining' | 'Success' | 'Fail' | 'Exception'
+
 export interface TransactionStatus {
-  status: 'None' | 'Mining' | 'Success' | 'Fail' | 'Exception'
-  errored: boolean
+  status: TransactionState
   transaction?: TransactionResponse
   receipt?: TransactionReceipt
   chainId?: ChainId
   errorMessage?: string
 }
+
+export function transactionErrored(transaction: TransactionStatus) {
+  return 'errorMessage' in transaction
```

Comment:
This seems good

Reply:
This function is basically noop now, errorMessage is always in TransactionStatus.

Reply:
A better way to check it now would be `transaction.errorMessage !== undefined`, but both works.

Reply:
@nezouse 
True about the helper, "one helper, more helper"... 

It might be just good to allow the users define helpers in a way it fits their various projects, coz some would even eventually still provide a helper around this helper because they want to use it in a class or library they use everywhere in their app...

This project is cool being simple, yet extracting the hardwork of interfacing with ethers.js directly to a hook...

Reply:
Fair enough. But one don't need to use helper :P